pkg/k8shandler/kibana/consoleexternalloglink.go

```diff
@@ -42,8 +40,9 @@ func (clusterRequest *KibanaRequest) RemoveConsoleExternalLogLink(resourceName s
 	)
 
 	err = clusterRequest.Delete(consoleExternalLogLink)
-	if err != nil && !errors.IsNotFound(err) {
-		return fmt.Errorf("Failure deleting %v ConsoleExternalLogLink %v", resourceName, err)
+	if err == nil || apierrors.IsNotFound(kverrors.Root(err)) {
```

Comment:
for future PRs -- can we have changes that are formatting in a secondary commit?

Reply:
I had to change the name of the package here